### PR TITLE
add get Docker container names flow + test

### PR DESCRIPTION
**Pull request by Bonczidai**
Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

**Comment by meirwah** [comment]
Please update @Bonczidai  ...

**Comment by Bonczidai** [comment]
updated

**Comment by meirwah** [comment]
test for names?

**Comment by raskin-aleksandr** [comment]
looks good to me
@oritstone can you take a look to

File: content/io/cloudslang/docker/containers/get_container_names.sl [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            - command:
+                default: >
+                  'docker ps' + ps_parameters
+                overridable: false
+            - pty: "'false'"
+            - username
+            - password:
+                required: false
+            - arguments:
+                default: "''"
```
Comment (oritstone) [inline content/io/cloudslang/docker/containers/get_container_names.sl]: We do not to pass empty strings as default values. You should not pass the arguments param at all to the ssh_flow

Comment (Bonczidai) [inline content/io/cloudslang/docker/containers/get_container_names.sl]: :+1:

**Pull request merged**